test/karma.conf.js

```diff
@@ -51,6 +51,8 @@ module.exports = function(config) {
       }
     },
     detectBrowsers: detectBrowsers,
+    // timeout after 2 minutes of no activity
```

Comment:
Experimenting increase of timeout per https://github.com/travis-ci/travis-ci/issues/8242#issuecomment-322199112